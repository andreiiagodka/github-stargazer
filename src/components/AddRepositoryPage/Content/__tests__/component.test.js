import { shallow } from 'enzyme'

import ContentComponent from '..'

describe('<ContentComponent />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ContentComponent />)

    expect(wrapper).toMatchSnapshot()
  })
})
