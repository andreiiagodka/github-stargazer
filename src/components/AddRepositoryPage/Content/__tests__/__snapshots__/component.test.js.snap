// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentComponent /> renders correctly 1`] = `
<Container
  fluid={true}
>
  <Col
    md={
      Object {
        "offset": 2,
        "span": 8,
      }
    }
  >
    <Connect(withRouter(WithFormik(AddRepositoryFormComponent))) />
  </Col>
</Container>
`;
